source/nodejs/adaptivecards-visualizer/src/containers/outlook.ts

```diff
@@ -120,8 +120,9 @@ export class OutlookContainer extends HostContainer {
         }
     }
 
-    public anchorClicked(rootCard: Adaptive.AdaptiveCard, anchor: HTMLAnchorElement): boolean {
+    public anchorClicked(element: Adaptive.CardElement, anchor: HTMLAnchorElement): boolean {
         var regEx = /^action:([a-z0-9]+)$/ig;
+        var rootCard = element.getRootElement();
```

Comment:
Should check that this is actually an AdaptiveCard instance.

Reply:
making var rootCard = element.getRootElement() as AdaptiveCard works?

Reply:
Yes that should be enough given there is a `if (rootCard)` statement after.